packages/nimble-components/src/table/tests/table.pageobject.ts

```diff
@@ -112,6 +113,17 @@ export class TablePageObject<T extends TableRecord> {
         );
     }
 
+    public getRenderedCellAnchor(
+        rowIndex: number,
+        columnIndex: number
+    ): Anchor {
+        const anchor = this.getRenderedCellView(
+            rowIndex,
+            columnIndex
+        ).shadowRoot!.querySelector(anchorTag);
+        return anchor as Anchor;
```

Comment:
Feels like we should be more robust here and not assume that the provided parameters will actually retrieve an `Anchor` element. I'd return an `Error` if the retrieved element is, in fact, not an `Anchor`.

Reply:
Done